第二階段 Guide - Part 1

js150422 opened the pull request:
姓名: 蔡宜珍
Discord:  jane08
作業網址:
http://18.210.228.7:3000/api/attractions?page=1
http://18.210.228.7:3000/api/attractions?page=0&keyword=%E8%97%9D%E8%A1%93
http://18.210.228.7:3000/api/attraction/100

麻煩您了^^

cwpeng reviewed (changes requested):
Issues should be fixed:
1. "nextPage" of http://18.210.228.7:3000/api/attractions?page=4 and http://18.210.228.7:3000/api/attractions?page=0&keyword=%E8%97%9D%E8%A1%93 should be null, because they are final pages.
2. http://18.210.228.7:3000/api/attractions?page=6 should response nothing in "data" because it's over max page.
3. "img" field in your response contains all image URLs, it's not correct. "img" field in every attraction should only contains image URLs for specific attraction.

Ask me to review again after you fix issues.

js150422 commented:
姓名: 蔡宜珍
Discord: jane08


Issues had repaired":
1.fixed(change Change loop logic)
 http://18.210.228.7:3000/api/attractions?page=4
 http://18.210.228.7:3000/api/attractions?page=0&keyword=%E8%97%9D%E8%A1%93

2.fixed(change Change loop logic)
http://18.210.228.7:3000/api/attractions?page=6

3.Img is fixed about one attraction url.

cwpeng commented:
Request me review again from your right side in this page, and you have to resolve conflicts as shown in this page.
